network_modeling/configs/templates/1.5/aruba/common/acl.j2

```diff
@@ -5,6 +5,7 @@ access-list ip mgmt
 {% set sequence = sequence+10 %}    {{ sequence }} permit tcp {{ variables.HMN_NETWORK_IP }}/{{ variables.HMN_NETMASK }} any eq https
 {% set sequence = sequence+10 %}    {{ sequence }} permit udp {{ variables.HMN_NETWORK_IP }}/{{ variables.HMN_NETMASK }} any eq snmp
 {% set sequence = sequence+10 %}    {{ sequence }} permit udp {{ variables.HMN_NETWORK_IP }}/{{ variables.HMN_NETMASK }} any eq snmp-trap
+{% set sequence = sequence+10 %}    {{ sequence }} permit udp any eq ntp {{ variables.HMN_NETWORK_IP }}/{{ variables.HMN_NETMASK }}
```

Comment:
I think there is an issue here as the `ntp.j2` template uses the NMN IPs for the `ntp server` definitions but this ACL list permits NTP via the HMN.

We can test this during our call.